data/json/items/ammo.json

```
@@ -788,7 +788,7 @@
     "category": "spare_parts",
     "price": 1000,
     "price_postapoc": 250,
-    "name": { "str": "copper" },
+    "name": { "str_sp": "copper" },
```

```suggestion
    "name": { "str": "copper" },
    "name": { "str_pl": "scraps of copper" },
```

---

The description and "count" field seem to be indicating that it's already multiple scraps. Your suggestion, BTW, would be done as:
```suggestion
    "name": { "str": "copper", "str_pl": "scraps of copper" },
```

---

Sure, but when you pop open your inventory and you see `5 copper` on the ground, it's not entirely clear what that means. 5 copper pieces, like this is D&D? Sheets of copper?

---

Below, I've suggested simply "str": "copper scrap".

---

Oh. And I've also changed the description from "copper scraps" to "copper bits", to avoid confusion with "scrap copper".